src/store/middlewares/account.js

```diff
@@ -220,6 +221,14 @@ const accountMiddleware = store => next => (action) => {
     case actionTypes.accountLoggedOut:
       store.dispatch(emptyTransactionsData());
       break;
+    case actionTypes.settingsUpdated: {
+      const tokensList = action.data.token && action.data.token.list;
+      const token = tokensList && Object.keys(tokensList)[0];
```

Comment:
`[0]` is quite hacky here and works only because we define 'BTC' before 'LSK'`.

```suggestion
      const token = tokensList && Object.keys(tokensList).find(token =>  tokensList[token]);
```

should work even if we have more tokens and enabling not just the first one.